vignettes/spatialHotspot.Rmd

```diff
@@ -0,0 +1,102 @@
+---
+title: "Spatial Data and Hotspot"
+package: "`r BiocStyle::pkg_ver('BiocStyle')`"
+output: BiocStyle::html_document
+vignette: >
+  %\VignetteIndexEntry{Spatial Data and Hotspot}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+# Preliminaries
+
+If you have yet to install VISION, we recommend installing the package from Github to install this package. Full source code can be found at the VISION Github repository, available [here](http://www.github.com/YosefLab/VISION).
```

Comment:
Same as the previous comment -- let's include a brief description of how to install Hotspot. Also could be useful to describe what Hotspot does.

Reply:
_Especially_ since this is the Hotspot demo, please flesh out the intro to Hotspot. Add a link to the documentation site. Add context for what this analysis and notebook is supposed to do.